vispy/visuals/filters/mesh.py

```diff
@@ -187,8 +190,11 @@ def _attach(self, visual):
     }
     normal = normalize(normal);
 
-    vec3 light_vec = normalize(v_light_vec);
-    vec3 eye_vec = normalize(v_eye_vec);
+    // Note (AK): Below is a way to flip the surface normal so that it's always in
+    // the view direction. However, the eye_vector is calculated as if the eye is
+    // a rectangular plane, not a point, causing it to "look around objects" a bit,
+    // and this gives rise to artifacts if we apply the below.
+    //normal = -faceforward(-normal, normal, eye_vec);
```

Comment:
Ah ok so this is the plane.

I'm not sure we want to systematically orient normals towards the viewer. This might correct some issues with inconsistent face orientations of the input models (#2069), but in specific cases only. For example, for face culling to work, there is no other possibility than to input a geomertry with a consistent definition.

Reply:
This description is also not correct. To be honest I am not sure why the artifacts appear.